src/vg/civcraft/mc/civmenu/TOSListener.java

```diff
@@ -20,67 +15,67 @@
 import net.md_5.bungee.api.chat.ClickEvent;
 
 public class TOSListener implements Listener {
-	
+
 	private CivMenu plugin = CivMenu.getInstance();
-	
+
 	public TOSListener() {
-		
+
 	}
-	
+
 	@EventHandler
 	public void playerJoinEvent(PlayerJoinEvent event) {
 		Player p = event.getPlayer();
-		if (!TOSManager.registeredPlayers
-				.containsKey(p.getUniqueId())) {
+		if (!TOSManager.registeredPlayers.containsKey(p.getUniqueId())) {
 			sendTOS(p);
 		}
 	}
-	
+
 	@EventHandler
-	public void playerMoveEvent(PlayerMoveEvent event){
+	public void playerMoveEvent(PlayerMoveEvent event) {
 		Location from = event.getFrom();
         Location to = event.getTo();
 
         if (from.getBlockX() == to.getBlockX()
-                && from.getBlockY() == to.getBlockY()
                 && from.getBlockZ() == to.getBlockZ()
                 && from.getWorld().equals(to.getWorld())) {
             // Player didn't move by at least one block.
             return;
         }
+		
 		Player p = event.getPlayer();
 		if (!TOSManager.registeredPlayers.containsKey(p.getUniqueId())) {
 			p.sendMessage(ChatColor.RED + "You must accept the terms in order to play.");
 			sendTOS(p);
-			event.setCancelled(true);
+			event.setTo(event.getFrom());
```

Comment:
For this line I purposely did this.  We don't want the player able to do anything until they sign.